app/templates/events/trackVolunteers.html

```diff
@@ -145,4 +145,30 @@ <h5 class="modal-title" id="exampleModalLabel">Add a volunteer</h5>
     </div>
   </div>
 </div>
+<!-- Modal for Add Past Volunteers-->
+<div class="modal fade float-start" id="addPastVolunteerModal" tabindex="-1" aria-labelledby="addPastVolunteerModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <form action="/addVolunteersToEvent/{{event.id}}" method="POST">
+          <div class="modal-header">
+            <h5 class="modal-title">Add Past Volunteers</h5>
+            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+          </div>
+              <div class="modal-body pl-5">
+                  {%for participant in recurringVolunteers%}
+                    <table>
+                      <input class="m-2" type="checkbox" name="volunteer[]" value= {{participant}} />
+                      <label class="form-check-label" for="PastVolunteers"> {{participant.firstName}} {{participant.lastName}}  </label>
```

Comment:
When adding an ID I run into the same problem where when I click the label for any checkbox after the first one that's generated It will only toggle the first checkbox.

Reply:
There can only be one unique id per page, so you would need to generate an id that is different for each item in the loop. Usually some fixed text + something that changes, like the username